src/NUnitConsole/nunit3-console/EnginePackageSettings.cs

```diff
@@ -101,7 +103,21 @@ public static class EnginePackageSettings
         /// are strings like "net-4.5", "mono-4.0", etc. Default is to
         /// use the target framework for which an assembly was built.
         /// </summary>
-        public const string RuntimeFramework = "RuntimeFramework";
+        [Obsolete("Use 'RuntimeFramework.RequestedRuntimeFramework' instead.")]
```

Comment:
```suggestion
        [Obsolete("Use 'RequestedRuntimeFramework' instead.")]
```

This isn't qualified, is it?

Reply:
No idea how I got that in there !